todo-app/src/Components/AddTodoItem.js

```diff
@@ -0,0 +1,99 @@
+//jshint esnext: true
+
+import React, { Component } from 'react'
+
+import Button from './Button';
+import InputField from './InputField';
+import './styles/addTodoItem.css';
+
+export default class AddTodoItem extends Component {
+
+    state = {
+        adding: false,
+        toBeAddedItem: '',
+        toBeAddedDeadline: ''
+    }
+
+
+
+    toggleAddFrom = event => {
+        this.setState({ adding: !this.state.adding });
+
+        //clearing fields for if it's a cancel
+        this.setState({
+            toBeAddedDeadline: '',
+            toBeAddedItem: ''
+        });
+    }
+
+   handleChangeTodoItem = event => {
+       this.setState({ toBeAddedItem: event.target.value });
+   }
+
+   handleChangeDeadline = event => {
+        this.setState({ toBeAddedDeadline: event.target.value })
+   }
+
+   handleSubmit = event => {
+       event.preventDefault();
+   }
+
+
+   handleAddTodo = () => {
+    const { toBeAddedItem, toBeAddedDeadline } = this.state;
+    this.props.handleAddTodo(toBeAddedItem, toBeAddedDeadline);
+    this.setState({
+        toBeAddedDeadline: '',
+        toBeAddedItem: ''
+    });
+    this.setState({ adding: false })
```

Comment:
You might want to clear the form here as well? Perhaps it would be worth creating a `clearForm` function to reuse throughout.